src/api/actions/checkBulkUploadJob.ts

```diff
@@ -0,0 +1,21 @@
+import { Maybe } from "../../utilities";
+import { UploadJobStatus } from "../../types";
+import { makeApiCall } from "./helpers";
+
+export const checkBulkUploadJob = async (
+  apiUri: string,
+  jobIds: string[]
+): Promise<UploadJobStatus[]> => {
+  let response: Maybe<UploadJobStatus[]>;
+  try {
+    response = await makeApiCall<UploadJobStatus[]>(
```

Comment:
Missed this earlier, but this is typed wrong, it's not an array that's returned.